pkg/chunkenc/symbols.go

```diff
@@ -310,14 +341,16 @@ func symbolizerFromCheckpoint(b []byte) *symbolizer {
 	db := decbuf{b: b}
 	numLabels := db.uvarint()
 	s := symbolizer{
-		symbolsMap: make(map[string]uint32, numLabels),
-		labels:     make([]string, 0, numLabels),
+		symbolsMap:      make(map[string]uint32, numLabels),
+		labels:          make([]string, 0, numLabels),
+		normalizedNames: make(map[uint32]string),
```

Comment:
nit: Preallocating this to `numLabels/2` is probably accurate enough and will be likely be less memory than the natural exponential growth Go does